README.md

```diff
@@ -20,9 +20,9 @@ A YouTube powered real-time music player built with AngularJS.
 
 ## Further work
 
-* ~~To implement drag and drop capabilities~~
-* Share playlist feature
-* Keyboard shortcuts for play, pause etc.
+[x] ~~To implement drag and drop capabilities~~
+[ ] Share playlist feature
+[ ] Keyboard shortcuts for play, pause etc.
 
```

Comment:
Not the correct way to represent a unordered list. Please check formatting again.